erpnext/manufacturing/doctype/production_plan/production_plan.js

```diff
@@ -116,34 +116,19 @@ frappe.ui.form.on('Production Plan', {
 	},
 
 	make_material_request: function(frm) {
-
-		frappe.confirm(__("Do you want to submit the material request"),
-			function() {
-				frm.events.create_material_request(frm, 1);
-			},
-			function() {
-				frm.events.create_material_request(frm, 0);
-			}
-		);
+		frm.events.create_material_request(frm, 1);
```

Comment:
frm.events.create_material_request(frm, 1)
Above code , passes the docstatus as "1" which Submitted , only then create material request via Create button/ Dashboard + option 
Below are the attached image for this fix: - 
![MR_not_saved](https://user-images.githubusercontent.com/80836439/117133657-28850900-adc2-11eb-9cd5-0131761bf862.PNG)
![MR_saved1](https://user-images.githubusercontent.com/80836439/117133660-29b63600-adc2-11eb-8d2f-5bf5e0933d2e.PNG)
![MR_saved](https://user-images.githubusercontent.com/80836439/117133663-29b63600-adc2-11eb-8176-90432b6cee95.PNG)

https://user-images.githubusercontent.com/80836439/117133665-2a4ecc80-adc2-11eb-9bf3-ce8aaa36fdd0.mp4